Dockerfile

```diff
@@ -21,11 +21,14 @@ RUN wget --quiet https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1
 RUN tar xvfj /opt/phantomjs-2.1.1-linux-x86_64.tar.bz2 && \
     ln -s /opt/phantomjs-2.1.1-linux-x86_64/bin/phantomjs /usr/bin/phantomjs
 
-# Copy the codebase
-COPY . /demo-test-suite
+RUN mkdir /demo-test-suite
```

Comment:
Maybe I'm getting the "wrong" result because of my local volume mounting, but I'd expect `COPY . /demo-test-suite` to copy the contents of the current directory into a /demo-test-suite directory. Shouldn't have anything to do with requirements.txt either way though.